src/vstest.console/TestPlatformHelpers/TestRequestManager.cs

```diff
@@ -80,7 +79,7 @@ public TestRequestManager()
                   TestRunResultAggregator.Instance,
                   TestPlatformEventSource.Instance,
                   new InferHelper(new AssemblyMetadataProvider()),
-                  MetricsPublisherFactory.GetMetricsPublisher(IsTelemetryOptedIn(), CommandLineOptions.Instance.IsDesignMode))
+                  MetricsPublisherFactory.GetMetricsPublisher(false, CommandLineOptions.Instance.IsDesignMode))
```

Comment:
nit: Good to comment this, why we always pass false. Hence by default testplatform will not publish anything.